auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -94,6 +96,52 @@ public static CheckEmailFragment getInstance(@NonNull FlowParameters flowParamet
         return fragment;
     }
 
+    public static void checkAccountExists(Activity activity,
+                                          final BaseHelper helper,
+                                          final CheckEmailListener listener,
+                                          @NonNull final String email,
+                                          @Nullable final String name,
+                                          @Nullable final Uri photoUri) {
```

Comment:
The refactor came out a little messier than I would have liked, but at least we're sharing the same logic.